doc/source/train/examples.yml

```diff
@@ -64,30 +55,39 @@ examples:
     contributor: community
     link: examples/hpu/bert
 
-  - title: Fine-tune of Stable Diffusion with DreamBooth and Ray Train
+  - title: Train a text classifier with DeepSpeed
+    frameworks:
+      - deepspeed
+      - pytorch
+    skill_level: intermediate
```

Comment:
Should this be intermediate or beginner? @woshiyyya

Reply:
I prefer it to be intermediate.